lib/jekyll/theme_builder.rb

```diff
@@ -0,0 +1,108 @@
+class Jekyll::ThemeBuilder
+  SCAFFOLD_DIRECTORIES = %w{
+    _layouts _includes _sass
+  }.freeze
+
+  attr_reader :name, :path
+
+  def initialize(theme_name)
+    @name = theme_name.to_s.gsub(/ /, "_").gsub(/_+/, "_")
+    @path = Pathname.new(File.expand_path(name, Dir.pwd))
+  end
+
+  def create!
+    create_directories
+    create_gemspec
+    create_accessories
+    initialize_git_repo
+  end
+
+  private
+
+  def root
+    @root ||= Pathname.new(File.expand_path("../", __dir__))
```

Comment:
`@root ||= Pathutil.new("../").expand_path(__dir__)` you should be able to do the same thing with Pathname.